app/views/shared/_header.html.slim

```diff
@@ -0,0 +1,28 @@
+head
+    = stylesheet_link_tag    "application", media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag "application", 'data-turbolinks-track' => true
+    = csrf_meta_tags
+
+    meta charset="utf-8"
+    meta http-equiv="X-UA-Compatible" content="IE=edge"
+    meta name="viewport" content="width=device-width, initial-scale=1"
+    meta name="description" content=""
+    meta name="author" content=""
+
+    title Asset Manager
+
+    / Bootstrap Core CSS
+    link href="css/bootstrap.min.css" rel="stylesheet"
```

Comment:
You are already including all stylesheets and javascript files in the asset pipeline. You shouldn't need these statements here.